wiki/_Sidebar.md

```diff
@@ -0,0 +1,55 @@
+**Core documentation**
+  * [Oppia's mission](https://github.com/oppia/oppia-android/wiki/Our-Mission)
+  * [Code of Conduct](https://github.com/oppia/oppia-android/blob/develop/.github/CODE_OF_CONDUCT.md)
+  * **[[Get involved!|Home]]**
+    * [Coders](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android)
+  * [Teams at Oppia Android](https://github.com/oppia/oppia-android/wiki/Teams-at-oppia-android)
+
+---
+**Developing Oppia**
+  * Contributing to Oppia Android
+    * [[Getting Started|Contributing-to-Oppia-android]]
```

Comment:
Is using the ``[[`` syntax still correct here?

Reply:
Yes, we cannot preview it in the PR. However, once it is deployed on the wiki it is viewable.

Reply:
Do we need to use that syntax here & elsewhere? I suspect it'd be less confusing for the pages to render correctly both when viewed directly on GitHub & when on the wiki (though this matters less for the sidebar, but it's also harder to verify the links are correct during review time).

Reply:
We're working on a GitHub Action to lint check such Mediawiki links: https://github.com/oppia/oppia-web-developer-docs/issues/21
you can instead use the traditional URLs `[Link Text](full-URL-of-wiki-page)` if you prefer the pages to be rendered.

Reply:
@gp201 but those don't work when looking at the .md file in GitHub code, right? E.g. see https://github.com/oppia/oppia-web-developer-docs/blob/develop/_Sidebar.md -- we might want to just use full links as a matter of course.

Reply:
## How does GitHub render various links?
|Types of links				|GitHub Render																									|Raw text																|Syntax										|
|---						|---																											|---																	|---										|
|Mediawiki with a link text	|![image](https://user-images.githubusercontent.com/30050862/225428075-2c4c2869-83d7-4f4e-bed1-4bec62734efa.png)|```[[Get involved!\|Home]]```													|`[[nameofwikipage\|Link-Text]]`				|
|Mediawiki with a URL link	|![image](https://user-images.githubusercontent.com/30050862/225428579-572dbe47-ae39-473c-9323-db292516f8a9.png)|```[[Create a Lesson!\|https://github.com/oppia/oppia/wiki/Lesson-Creation-Guide]]```|`[[nameofwikipage\|full-URL-of-wiki-page]]`	|
|**Markdown Links**				|![image](https://user-images.githubusercontent.com/30050862/225427300-c8f739db-1078-4ca8-a344-f1c940d4589f.png)|`[Oppia's mission](https://github.com/oppia/oppia/wiki/Oppia's-Mission)`	|`[Link Text](full-URL-of-wiki-page)`|

You can use Markdown Link syntax if you prefer the links to be rendered correcctly.

## How to see how the page will render in wiki?
![image](https://user-images.githubusercontent.com/30050862/225425895-b8e736b1-7de0-4c83-892a-d0cf3212ab66.png)
These options present at top right corner of every file in the PRs

- `Left option`: **Display the source diff**: This will allow you to see the raw markdown text
- `Right option`: **Display the rich diff**: This will allow you to see how the page will render in wiki

Reply:
Migrated all Media wiki links to markdown links

Reply:
Thanks @gp201!